packages/blockchain-wallet-v4-frontend/src/data/components/brokerage/types.ts

```diff
@@ -188,81 +186,3 @@ export type BrokerageState = {
   fiatCurrency: WalletFiatType | undefined
   redirectBackToStep: boolean
 }
-
-interface FetchBankLinkCredentialsType {
```

Comment:
This was very redundant boilerplate that we needed, but since reducers/actions/actionTypes are created by RTK, we can just delete these as the library handles all the proper types!